backend/apps/core/locks.py

```
@@ -0,0 +1,140 @@
+        except Exception as e:
+            logger.error(f"Failed to force release lock {lock_key}: {e}")
+            return False
+
+    @staticmethod
+    def get_all_locks() -> list:
+        try:
+            from django_redis import get_redis_connection
+            redis_client = get_redis_connection("default")
+            keys = redis_client.keys("celery_lock:*")
+            return [k.decode('utf-8') for k in keys]
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)